elliottlib/cli/__main__.py

```diff
@@ -248,21 +248,20 @@ async def verify_payload(ctx, payload, advisory):
         # The machine-os-content image doesn't follow the standard
         # pattern. We need to skip that image when we find it, it is
         # not attached to advisories.
-        if 'com.coreos.ostree-commit' in labels:
-            yellow_prefix("Skipping machine-os-content image: ")
+        if 'com.coreos.ostree-commit' in labels or 'rhel-coreos' in image_name:
```

Comment:
Hardcoding the tag name is not a good idea. From 4.12, we have multiple RHCOS images in the payload.

RHCOS tag names can be found from group config, see https://github.com/openshift/ocp-build-data/blob/241943cb506c875121a5df44aed67728b809dceb/group.yml#L47

Reply:
Ah didn't know that existed, thanks I'll get that in

Reply:
Done now